.travis.yml

```diff
@@ -26,3 +32,12 @@ notifications:
       - cc-devel@lists.01.org
     on_success: change
     on_failure: always
+
+addons:
+  coverity_scan:
+    project:
+      name: "01org/cc-oci-runtime"
+      description: "Build submitted via Travis CI"
+    build_command_prepend: "./autogen.sh --enable-cppcheck --enable-valgrind; make clean"
+    build_command: "make -j 4"
+    branch_pattern: coverity_scan
```

Comment:
So is the plan to push a coverity branch every day as part of the daily release process?